CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The following is a curated list of changes in the Enact moonstone module, newest
 
 ### Fixed
 
+- `moonstone/VirtualList.VirtualList`, `moonstone/VirtualList.VirtualGridList` to support navigation with spottable children inside an item
```

Comment:
Can't we shorten this as `moonstone/VirtualList`?

Reply:
Done!